mypy/sharedparse.py

```diff
@@ -90,3 +92,7 @@
 
 def special_function_elide_names(name: str) -> bool:
     return name in MAGIC_METHODS_POS_ARGS_ONLY
+
+
+def argument_elide_name(name: Union[str, Tuple, None]) -> bool:
```

Comment:
The type of the `name` argument is funny. It seems that the fast parser does some unsafe things and doesn't always conform to type annotations. Anyway, this is an issue with the fast parser only.